### ISSUE TITLE
nothing else shows

### ISSUE THREAD

**Issue by SmokeyPlayz** [jhdygpd/Gamek989-roblox-group-finder-ratelimit-bypass#4]
![image](https://github.com/jhdygpd/Gamek989-roblox-group-finder-ratelimit-bypass/assets/80928946/c3afe691-4441-4430-b00a-7f044f4acbf8)

**Comment by jhdygpd** [comment]
yes, because to accualy find anything you need like 20 cookies and log shows every 2 min

**Comment by jhdygpd** [comment]
Webhook message id
